core/unix/inc/TUnixSystem.h

```diff
@@ -216,7 +216,6 @@ class TUnixSystem : public TSystem {
    int               GetMemInfo(MemInfo_t *info) const override;
    int               GetProcInfo(ProcInfo_t *info) const override;
 
-   ClassDefOverride(TUnixSystem,0)  //Interface to Unix OS services
```

Comment:
I am confused.  This means that at run-time we can no longer distinguish between `TSystem` and `TUnixSystem` `(at least not via Core/Meta interfaces).   What is the motivation for this changes?